redash/query_runner/big_query.py

```diff
@@ -101,6 +101,10 @@ def configuration_schema(cls):
                     'type': 'string',
                     'title': 'Project ID'
                 },
+                'additionalProjects': {
```

Comment:
the change here is only for the big query backend service - what about the data source config update page (separate PR?)

Reply:
There isn't a datasource config page for big query, or any other datasource, that's all the config available, you can't validate the config, change the html controls, or anything of this sort in the current way redash is designed, only add and remove fields of primitive types.